wrappers/python/nntile/nntile_core/starpu.pyi

```diff
@@ -0,0 +1,18 @@
+class Config:
```

Comment:
Add a header to the file. You can copy header from another Python file.

Reply:
Why?

Reply:
A header like
```
# @copyright (c) 2022-2023 Skolkovo Institute of Science and Technology
#                           (Skoltech). All rights reserved.
#
# NNTile is software framework for fast training of big neural networks on
# distributed-memory heterogeneous systems based on StarPU runtime system.
#
# @file wrappers/python/nntile/pipeline.py
# TRaining pipeline of NNTile Python package
#
# @version 1.0.0
# @author Aleksandr Mikhalev
# @author Aleksandr Katrutsa
# @date 2023-09-20
```
tells a reader that the file is a part of NNTile project with its location. Sometimes files are just copied from a project to project without any `git` history. Probably, list of authors and date of last update are an overkill, but altogether such an information helps external collaborators.